bucket/mariadb.json

```diff
@@ -59,8 +59,8 @@
     ],
     "persist": "data",
     "checkver": {
-        "url": "https://downloads.mariadb.org/",
-        "regex": "Download ([\\d.]+) Stable"
+        "url": "https://downloads.mariadb.org/rest-api/mariadb/all-releases/",
+        "regex": "MariaDB Server ([\\d.]+)\""
```

Comment:
Need to use JSONPath here

https://github.com/ScoopInstaller/Main/blob/master/bucket/haskell.json#L33

Reply:
Check if this works

$.releases[?(@.status=='stable')].text

This worked for me. Tested at https://jsonpath.com/ but am not sure if Scoop would support this

Reply:
Or $.releases[?(@.status=='stable')].release_number to get the release_number directly

Reply:
This is most convenient approach here, the jsonpath tested above doesn't work here and I don't know why, and that's why I use regex.

Reply:
BTW, Scoop support jsonpath regex (https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-jsonpath-in-checkver), but debug is difficult.

Reply:
Will create a PR to fix this jsonpath issue today.